tests/config/jest-light-runner/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@babel/jest-light-runner",
```

This is not under `@babel`
```suggestion
  "name": "jest-light-runner",
```

---

Maybe you missed this readme file https://github.com/fisker/prettier/blob/esm-format-test/tests/config/jest-light-runner/readme.md

It's copied from babel, let's not do unnecessary change, so we can keep sync with the one in babel repo.

---

I saw the README but It seemed a bit high-context that a package named `@babel/...` is defined by Prettier.

If there are no changes between all to the jest-light-runner implemented in Babel and Prettier, can we have a script to synchronize it? And if the README says something like the following about the script, I think it's OK.

> Do not update this package manually. This package will only be updated by the script.

---

Of course, that work can be done in other PRs.

---

I've made some changes... I don't have strong feeling for the package name, I can change it if you insist. 

One thing you should know, these files even not following the code style, I turned eslint and prettier off on them.

---

If there are any changes, the name of the package should not start with `@babel/`.

Btw, how did you change the package?

---

> Btw, how did you change the package?

https://github.com/prettier/prettier/pull/11902/commits/baabfda5fcad07057f1b7fd417b9c2f076d1500a#diff-0688392cb01e5b47f4f1249386e343f205a7390c18506421c2442616a22a2fddR29

L29-L36 was added.

---

Thanks.

---

https://github.com/prettier/prettier/pull/11902/commits/172153127b6134f7bd9ce52f77774c9409eae191